src/AuthProvider.tsx

```diff
@@ -202,7 +202,17 @@ export const AuthProvider = (props: AuthProviderProps): React.JSX.Element => {
                                 try {
                                     return await userManager[key](args);
                                 } catch (error) {
-                                    dispatch({ type: "ERROR", error: error as Error });
+                                    dispatch({
+                                        type: "ERROR",
+                                        error: {
```

Comment:
Can't we re-use here `normalizeErrorFn`?

handled in `normalizeErrorFn`:
- name: nameOf(error),
- message: messageOf(error, fallbackMessage),
- innerError: error,
- stack: stackOf(error, true),
- source: source,

missing:
- args, which can be extended in `normalizeErrorFn` to passed as well...

This would hide boilerplate and duplication code and we would not have export `nameOf`, `messageOf` and `stackOf`...

Reply:
I've adjusted, but not fully. I believe the current approach is more simple.